code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3289,6 +3289,47 @@ datum
 			thirst_value = 1
 			bladder_value = -1
 
+		fooddrink/turmeric
+			name = "turmeric powder"
+			id = "turmpowder"
+			description = "Has a warm, spicy scent and nausea-soothing properties. But if you get this shit on something, the stain's NEVER coming out."
+			reagent_state = SOLID
+			fluid_r = 224
+			fluid_g = 168
+			fluid_b = 12
+			transparency = 255
+
+			reaction_mob(var/mob/M, var/method = TOUCH, var/volume)
+				. = ..()
+				if(method == TOUCH && volume >= 10)
+					boutput(M, "<span class='notice'><b>The chemical stains your skin!</b></span>")
+					M.color = rgb(rand(175,255),rand(110,169),92)
+					return
+
+			reaction_obj(var/obj/O, var/volume)
+				if(volume >= 10)
+					O.color = rgb(rand(175,255),rand(110,169),92)
+					return
+
+			reaction_turf(var/turf/T, var/volume)
+				if(volume >= 20)
+					T.color = rgb(rand(175,255),rand(110,169),92)
+					return
```

Comment:
Better to use color matrices for coloring stuff, a la paint.
Example of such in `/datum/statusEffect/dripping_paint`